code/modules/client/verbs/viewport.dm

```diff
@@ -0,0 +1,48 @@
+/// Used in OnResize() - Holds the clients prefs.
+GLOBAL_VAR_INIT(lock_client_view_x, null)
+GLOBAL_VAR_INIT(lock_client_view_y, null)
```

Comment:
are these debug vars?

Reply:
Admit setable constraints for debugging or future anti-lag measures if they're ever made. I don't remember FULLY myself, but that's what I recon I did.